### DISCUSSION TITLE
need help setting up self-hosting

### DISCUSSION THREAD

**Discussion by thevegasmovement** [Budibase/budibase#7236] [Help] [answered]
I already went down the rabbithole with trying to set this up last year as self-hosting and got nowhere. 

I literally need someone to do it for me. I have a digital ocean account. I need someone who can set this up for me please!!

**Comment by shogunpurple** [comment, marked as answer]
@thevegasmovement if you are not comfortable managing and running your own self hosted software, and unless you have specific needs around self hosting, you would be much better served using budibase cloud. Self hosting budibase is not just a case of setting it up - you will need to maintain it, perform updates, backups, and debug issues that may happen on your hardware, which generally requires terminal knowledge.

Since last year, we've introduced a ton of different methods for running budibase self hosted, including a one click droplet for digitalocean. More information here:
https://docs.budibase.com/docs/hosting-methods

**Reply by thevegasmovement** [reply]
ty. My biggest issue is that Martin had given me 10k automations/month last year that I no longer have in my account (it was the desktop app and I am not seeing that i have that anymore) and I need more than 1000 rows. I would instead be willing to pay for more apps, more rows, more automations (if I cannot get my 10k automations and an increase in the others) as well. Please advise? *I do have a copy of the discord conversation where I was given those automation increases

**Reply by shogunpurple** [reply]
@thevegasmovement they were given when we were in alpha - it wasn't forever. The desktop app is no longer a thing since we are completely web based now. We plan on releasing our pricing tiers at the end of this month, where you will be able to upgrade your account and get access to more automations.

**Reply by thevegasmovement** [reply]
ok perfect ty - I will be on the lookout for that. 
also can you please remind me if ther was an eta on "own records" or if it is just something that will come in the future?
